### PR TITLE
Addresses various parameter issues

### PR DESCRIPTION
**Pull request by jamesmosys**
#131 #133 #135 #137

### CODE REVIEW

File: src/main/python/camdkit/model.py [hunk 1]
```diff
@@ -1345,9 +1345,9 @@ def make_json_schema() -> dict:
     }
 
 class LensCustom(ArrayParameter):
-  """Until the OpenLensIO model is finalised, this list provides custom
-  coefficients for a particular lens model e.g. undistortion, anamorphic
-  etc
+  """This list provides optional custom additonal coefficients for a
+  particular lens model. The meaning of which would require negotiation
+  between a particular producer and consumer.
```
Comment (JGoldstone) [inline src/main/python/camdkit/model.py:1341]: ...lens model, the meaning of...

I will raise this in the main slack, but given that people might have, e.g., custom accelerometer data, would this be better named `LensCustomDistortion`?

Comment (jamesmosys) [reply src/main/python/camdkit/model.py:1341]: It's not necessarily describing just distortion though - there could be other characteristics of the lens that a particular producer/consumer pair want to share... I'll clarify the comment...

**Review by JGoldstone — APPROVED** [review]
Looks good to me.

**Review by JGoldstone — COMMENTED** [review]
LGTM

**Review by JGoldstone — COMMENTED** [review]
LGTM

**Review by JGoldstone — COMMENTED** [review]
LGTM

**Review by JGoldstone — COMMENTED** [review]
LGTM as far as it goes, but in future I think we should add the ID field to the set of fields exercised in our unit tests

**Review by JGoldstone — COMMENTED** [review]
LGTM

**Review by JGoldstone — COMMENTED** [review]
If this is where we leave it, it's ... OK. It's hard to anticipate what people might do with a custom field, basically. But see my comments in the Slack channel on another way to carry custom lens data. If no one likes that idea, then it's fine to let this stand.

**Pull request merged**